githubdelivery/src/test/java/io/dockstore/githubdelivery/package-info.java

```diff
@@ -0,0 +1 @@
+package io.dockstore.githubdelivery;
```

Comment:
Would be good to document a new package, can use PR description including links to issues as a staring point.

Reply:
Added copyright; this is a directory in test path; the main src path already had a  package-info for the same package with some comments. I googled around a little and couldn't figure a best practice for this, but I don't think we should duplicate.